integration-tests/contract-register-global-constant.spec.ts

```diff
@@ -38,7 +38,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
     });
 
     test(
-      'Register a Micheline expression to the global table of constants with auto-estimation of the fee, storage limit and gas limit',
+      'Verify contract.registerGlobalConstant to register a global constant and deploy a contract with the constant with auto-estimation of the fee, storage limit and gas limit',
```

Comment:
~and deploy a contract with the constant~

Reply:
corrected